packages/flutter/lib/src/widgets/sheet.dart

```diff
@@ -0,0 +1,677 @@
+import 'dart:async';
+import 'dart:ui';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+
+const Curve _decelerateEasing = Cubic(0.0, 0.0, 0.2, 1.0);
```

Comment:
I am surprised that the widget layer has so much opinion on the visuals of the sheet. Shouldn't this be upto the design languages to configure?